ntoskrnl/cc/view.c

```diff
@@ -411,8 +412,7 @@ CcRosFlushDirtyPages (
         if (--SharedCacheMap->OpenCount == 0)
             CcRosDeleteFileCache(SharedCacheMap->FileObject, SharedCacheMap, &OldIrql);
 
-        if (!NT_SUCCESS(Status) && (Status != STATUS_END_OF_FILE) &&
-            (Status != STATUS_MEDIA_WRITE_PROTECTED))
+        if (!NT_SUCCESS(Status))
         {
             DPRINT1("CC: Failed to flush VACB.\n");
```

Comment:
Can you add the status to the debug print while you're at it? In its current state, it's just noise.

Reply:
Added.